features/FxA-54-connect-another-device-phase-3/README.md

```diff
@@ -0,0 +1,367 @@
+7. When opening Firefox Accounts, email address from step 1 should be filled in.
+8. Sign in with the password from step 1.
+
+###### Sign up in Firefox desktop, send link to Android, Firefox already installed.
+
+1. Using Firefox desktop, sign in to Sync by creating a new Firefox Account using
+   a mozilla or softvision address.
+2. In the same instance of Firefox desktop, open the verification link with
+   the `&signinCodes=true&forceExperimentGroup=treatment` query parameters.
+3. Enter a valid phone number for an Android device, submit.
+4. Open the SMS link on an Android device. The link will be of the form `https://<server>/m/:signinCode`, write down the `signinCode`.
+5. Firefox should open.
+6. Open a tab to `https://<server>/signin?context=fx_fennec_v1&service=sync&signin=<signinCode from step 4>`
+7. When opening Firefox Accounts, email address from step 1 should be filled in.
+8. Sign in with the password from step 1.
+
+##### Testing with support for deep links in Firefox for Android
+
+###### Sign up in Firefox desktop, send link to Android, Firefox not installed
+
+1. Using Firefox desktop, sign in to Sync by creating a new Firefox Account using
+   a mozilla or softvision address.
+2. In the same instance of Firefox desktop, open the verification link with
+   the `&signinCodes=true&forceExperimentGroup=treatment` query parameters.
+3. Enter a valid phone number for an Android device, submit.
+4. Open the SMS link on an Android device.
+5. Install Firefox from the Google Play store.
+6. Open Firefox, complete the "firstrun flow".
+7. Firefox Accounts should open, email address from step 1 should be filled in.
+8. Sign in with the password from step 1.
+
+###### Sign up in Firefox desktop, send link to Android, Firefox already installed.
+
+1. Using Firefox desktop, sign in to Sync by creating a new Firefox Account using
+   a mozilla or softvision address.
+2. In the same instance of Firefox desktop, open the verification link with
+   the `&signinCodes=true&forceExperimentGroup=treatment` query parameters.
+3. Enter a valid phone number for an Android device, submit.
+4. Open the SMS link on an Android device.
+5. Firefox should open to the firstrun flow. Complete the "firstrun flow".
+6. Firefox Accounts should open, email address from step 1 should be filled in.
+7. Sign in with the password from step 1.
+
+##### Reference Images
+
+###### Send SMS
+<img src="images/send_sms.png" width="250" alt="Send SMS"/>
+
+###### SMS received
+Image needed
+
+###### Install Firefox
+<img src="images/install_firefox_play_store_android.png" width="250" alt="Send SMS"/>
+
+###### Open Firefox
+<img src="images/open_firefox_play_store_android.png" width="250" alt="Send SMS"/>
+
+###### Firstrun
+Image needed
+
+##### Signin page with valid signinCode
+<img src="images/signin_valid_code_android.png" width="250" alt="Send SMS"/>
+
+#### iOS
+
+##### Testing without support for deep links in Firefox for iOS
```

Comment:
Again, Fx beta release channel supports deep linking. We should be able to remove this section since https://github.com/mozilla/fxa-content-server/pull/5221 provides us ways of installing Beta instead of Release.